bin/cumulus.rb

```diff
@@ -230,6 +231,13 @@ def self.cloudfront
       else
         cloudfront.sync_one(ARGV[2])
       end
+    elsif ARGV[1] == "invalidate"
+      if ARGV.size != 3
+        puts "Must specify one invalidation to run"
```

Comment:
I might put this statement into command form : `Specify one invalidation to run`